tools/clang/test/DXC/root_sig.test

```diff
@@ -27,7 +27,7 @@
 // Set root signature when already has one should succeed
 // RUN: %dxc %t.define.cso /dumpbin /setrootsignature %t.rootsig.cso /Fo %t.smoke.rsadded.cso
 // Make sure %t.smoke.rsadded.cso exist.
-// RUN: file -E %t.smoke.rsadded.cso
+// RUN: ls -l %t.smoke.rsadded.cso
```

Comment:
Why `-l`? I think `ls` by itself (or `ls -1` if we're checking the output maybe) is sufficient.

Reply:
Change to only ls.